### PR TITLE
refactor(ui): add onPush strategy on workflow view

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

File: ui/src/app/views/workflow/run/node/pipeline/spawninfo/spawninfo.component.ts [hunk 1]
```diff
@@ -125,16 +139,19 @@ export class WorkflowRunJobSpawnInfoComponent implements OnDestroy {
...
-                    if (serviceSpawnInfos && serviceSpawnInfos.length > 0) {
-                        this.spawnInfos = this.getSpawnInfos(serviceSpawnInfos);
-                    }
-                    if (this.jobStatus === PipelineStatus.SUCCESS || this.jobStatus === PipelineStatus.FAIL ||
-                        this.jobStatus === PipelineStatus.STOPPED) {
-                        this.stopWorker();
-                        if (this.nodeJobRun.spawninfos && this.nodeJobRun.spawninfos.length > 0) {
-                            this.spawnInfos = this.getSpawnInfos(this.nodeJobRun.spawninfos);
+                    this.zone.run(() => {
+                        this._cd.markForCheck();
```
Comment (bnjjj) [inline ui/src/app/views/workflow/run/node/pipeline/spawninfo/spawninfo.component.ts]: should not be at the bottom ?

Comment (sguiheux) [reply ui/src/app/views/workflow/run/node/pipeline/spawninfo/spawninfo.component.ts]: I don’t think it matters, the current js thread has to end before that angular trigger change detection no?

Comment (bnjjj) [reply ui/src/app/views/workflow/run/node/pipeline/spawninfo/spawninfo.component.ts]: no idea

Comment (sguiheux) [reply ui/src/app/views/workflow/run/node/pipeline/spawninfo/spawninfo.component.ts]: ok so at the end

**Pull request merged**